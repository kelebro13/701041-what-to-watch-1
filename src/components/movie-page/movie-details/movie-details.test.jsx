import renderer from "react-test-renderer";
import {film} from "../../test/mock";
import MovieDetails from "./movie-details";

it(`renders properly`, () => {

  const tree = renderer
    .create(<MovieDetails film={film} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
