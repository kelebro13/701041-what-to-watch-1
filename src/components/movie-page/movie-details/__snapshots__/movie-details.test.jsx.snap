// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Wes Andreson
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        <span>
          Bill Murray, 
          <br />
        </span>
        <span>
          Edward Norton, 
          <br />
        </span>
        <span>
          Jude Law, 
          <br />
        </span>
        <span>
          Willem Dafoe, 
          <br />
        </span>
        <span>
          Saoirse Ronan
        </span>
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1h 39m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Comedy
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2014
      </span>
    </p>
  </div>
</div>
`;
